src/step-11-authorization/package.json

```diff
@@ -6,8 +6,6 @@
     "test": "c8 tap --no-cov"
   },
   "dependencies": {
-    "fastify": "^4.28.1",
-    "mercurius": "^14.1.0",
```

Comment:
These two were already installed in the workspace root. Better to avoid potential for duplicate versions in the future and centralize to the root package.json.

Reply:
yep agreed. hopefully there wasn't a reason why they were installed in the individual workspaces. thanks for catching it